wgpu-hal/src/gles/egl.rs

```diff
@@ -618,8 +619,13 @@ impl crate::Instance<super::Api> for Instance {
                 .unwrap();
             (display, Some(Arc::new(library)), WindowKind::AngleX11)
         } else {
-            log::info!("Using default platform");
-            let display = egl.get_display(egl::DEFAULT_DISPLAY).unwrap();
+            log::info!("No windowing system present. Using surfaceless platform");
```

Comment:
here we should check for extension before trying to use it

Reply:
Done